tests/unit/multi_select_test.coffee

```diff
@@ -1,36 +1,34 @@
-multiselect = null
-dispatcher = null
-
-module "[Unit] Multi-select unit tests",
-  setup: ->
-    Ember.run ->
-      dispatcher = Ember.EventDispatcher.create()
-      dispatcher.setup()
-      multiselect = Ember.Widgets.MultiSelectComponent.create
-        content: ['foo', 'bar', 'barca', 'baz']
-
-  teardown: ->
-    Ember.run ->
-      dispatcher.destroy()
-      multiselect.destroy()
+moduleForComponent 'multi-select', '[Unit] Multi select component'
 
 test 'Test preparedContent after some options are already selected', ->
   expect 2
+  multiselect = @subject
```

Comment:
`setup` is called for every test, so we were already creating one per test.

And that's usually a good pattern to avoid test pollution.